package.json

```diff
@@ -17,6 +17,7 @@
     "Valentin Lamatte <valentinl@forestadmin.com>",
     "Arnaud Besnier <arnaudibesnier@gmail.com>"
   ],
+  "repository": "https://github.com/ForestAdmin/lumber",
```

Comment:
[ Improvement ] As stated in the doc, repository has to be used with 'type' ("type" : "git"). If you want to use the shortcut (as you did) it's better to write that way : "repository": "github:ForestAdmin/lumber".

See ref : https://docs.npmjs.com/files/package.json#repository

Reply:
Thank you! 👍 I should have read the doc 😅